### ISSUE TITLE
Invalid ARIA attribute value: aria-activedescendant="sug-noResultsFound"

### ISSUE THREAD

**Issue by tdempsey2** [microsoft/fluentui#22246]
### Environment Information

- **Package version(s)**: (8.62.4)
- **Browser and OS versions**: (Chrome: Version 99.0.4844.84 Windows 11)

### Describe the issue:

[Microsoft Accessibility Insights for Web plugin] is reporting issues when we have a Dropdown component with pre-selected items opened.
`<input autocapitalize="off" autocomplete="off" aria-autocomplete="both" spellcheck="false" placeholder="e.g. Biology" aria-labelledby="majorsAutocomplete1469" class="ms-BasePicker-input input-161" id="combobox-id__1470" aria-activedescendant="sug-noResultsFound" aria-controls="" aria-expanded="false" aria-haspopup="listbox" role="combobox" data-lpignore="true" value="" aria-describedby="selected-items-id__1470" style="font-family:  #inherit;">`

A similar issue: #15383
### Please provide a reproduction of the issue in a codepen:

https://codepen.io/tiedde/pen/rNpwgpL
![image](https://user-images.githubusercontent.com/3530920/160724534-834ced24-39f9-4630-b575-a4ea37d71a56.png)
![image](https://user-images.githubusercontent.com/3530920/160724769-4204a321-05fe-45cb-beb8-9ac0af423c14.png)

According to the documentation, you are supposed to send an Id of an option of the dropdown/combobox`
Which we are doing. 

#### Actual behavior:

Accessibility validations fail for Combobox.

#### Expected behavior:

No Accessibility error.

### Documentation describing expected behavior

[Using aria-activedescendant to allow changes in focus within widgets to be communicated to Assistive Technology - WCAG WG (w3.org)](https://www.w3.org/WAI/GL/wiki/Using_aria-activedescendant_to_allow_changes_in_focus_within_widgets_to_be_communicated_to_Assistive_Technology#:~:text=The%20aria-activedescendant%20attribute%20is%20generally%20used%20in%20composite%2cthe%20current%20active%20child%20element%20within%20the%20widget.)

**Comment by micahgodbolt** [comment]
Your demo isn't currently working, and is nowhere near a reduced testcase. What's the easiest way for me to repro this bug?
